scripts/superdesk/editor2/editor.js

```diff
@@ -834,19 +834,25 @@ angular.module('superdesk.editor2', [
                         }
                     });
                 }
+                var debouncedInit = _.debounce(init, 250);
```

Comment:
it could, protractor can wait for all $timeouts and $http requests to finish, but it has no idea about lodash methods